### DISCUSSION TITLE
Google Summer of Code 2024

### DISCUSSION THREAD

**Discussion by bigfooted** [su2code/SU2#2220] [General]
**UPDATE March 19, 2024: GSOC Introduction phase is over, if you are still looking for a gsoc project by now, you are too late, sorry!**

SU2 has been approved as a GSoC organization for GSoC 2024!

Interested in working on SU2 in a Google Summer of Code project? SU2 is a mentor organization for GSoC '24. Here is some additional information for you.

What is Google Summer of Code?

https://summerofcode.withgoogle.com/

The basic idea is that you will work together with a mentor on a project. We have selected a couple of main topics for you but it is up to you to write a more complete project proposal. If you have ideas of your own, that is fine too but please find a mentor to supervise such a project. 

[GSoC projects link](https://docs.google.com/document/d/1HGeXqCF3AG6wBKJidaNiGxYly0lD_RBMk90zIGFVMyw/edit?usp=sharing)

If you would like to apply, please do the following:

Make sure that you have subscribed to CFD-online and github, and additionally join the developers team on slack at su2devteam.slack.com. If you are interested in applying, you have 4 options:
~~1. Send an email to gsoc@su2foundation.org~~
~~2. Go to slack and post in the gsoc subchannel of SU2 (email us to get an invitation to our slack workspace)~~
~~3. Post a message on cfd-online in the News & Announcement section with gsoc24 in the title: https://www.cfd-online.com/Forums/su2-news-announcements/~~
~~4. Start a discussion on github discussions with gsoc24 in the title: https://github.com/su2code/SU2/discussions~~

To Apply:

    Create a brief CV with your contact details, your education and code experience.
    Briefly write about your experience and interests in SU2 and the specific project you would like to work on.
    Briefly write about your current work schedule: are you studying/working, how will you manage the time, etc..
    Send it to one of the channels mentioned above. Note that slack is the most private option as it is a closed environment.

We will then couple you with a mentor. Then finally:

    Together with a mentor: create a project planning with a timeline containing a breakdown of the project in parts with periodic deliverables/milestones.

Note that applying does not mean acceptance into the program. We will carefully consider your application to see if you are capable of the job, taking into account your experience and availability.

**Comment by ankitmrj** [comment]
Greetings,
Respected Sir @bigfooted  ,

I am first year undergraduate student, of Computer Science at State University from India. I am thrilled to work on project " Feasibility of GPU acceleration in SU2".
I have completed  analysis related to adaptation of the existing SU2 code using NVBLAS. The NVBLAS Library is a GPU-accelerated Library that implements BLAS (Basic Linear Algebra Subprograms).

Please help me to learn and contribute more idea to this project.
How to get template for proposal ?.

Thanking You !
